dolphinscheduler-scheduler-plugin/dolphinscheduler-scheduler-quartz/src/main/java/org/apache/dolphinscheduler/scheduler/quartz/QuartzZeroSizeThreadPool.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.dolphinscheduler.scheduler.quartz;
```

Comment:
Please add apache license header for the new class file.

Reply:
fixed